From c5ee522bc7dbe419856f5f1ba37c2527b2974adc Mon Sep 17 00:00:00 2001
From: Eric Naim <dnaim@cachyos.org>
Date: Mon, 3 Feb 2025 12:58:18 +0800
Subject: [PATCH 3/3] drm: Remove driver data from struct drm_driver

This was removed in 6.14 from commit cb2e1c2136
("drm: remove driver date from struct drm_driver and all drivers")

Signed-off-by: Eric Naim <dnaim@cachyos.org>
---
 kernel-open/nvidia-drm/nvidia-drm-drv.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/kernel-open/nvidia-drm/nvidia-drm-drv.c b/kernel-open/nvidia-drm/nvidia-drm-drv.c
index d62c7057..5a92b1f4 100644
--- a/kernel-open/nvidia-drm/nvidia-drm-drv.c
+++ b/kernel-open/nvidia-drm/nvidia-drm-drv.c
@@ -87,6 +87,7 @@
 #endif
 
 #include <linux/pci.h>
+#include <linux/version.h>
 #include <linux/workqueue.h>
 
 /*
@@ -1915,7 +1916,10 @@ static struct drm_driver nv_drm_driver = {
     .name                   = "nvidia-drm",
 
     .desc                   = "NVIDIA DRM driver",
+
+#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 14, 0)
     .date                   = "20160202",
+#endif
 
 #if defined(NV_DRM_DRIVER_HAS_DEVICE_LIST)
     .device_list            = LIST_HEAD_INIT(nv_drm_driver.device_list),
-- 
2.48.1

